### DISCUSSION TITLE
`ifrs17sim` references info

### DISCUSSION THREAD

**Discussion by declann** [lifelib-dev/lifelib#72] [General]
Hi,

I know that this is an old-style project, but I'm still having a look to do some simplified csm modelling.

Comments include lots of references to lines and spreadsheet cells (apparently), e.g.

- ["""Insurance Revenue (82-85, B120-B125)"""](https://github.com/lifelib-dev/lifelib/blob/f5fe26c2577488683481c376900f18e95b1e2575/lifelib/projects/ifrs17sim/model/IFRS/__init__.py#L194-L195)
- ["""Transfer of services (44(e)->B119)](https://github.com/lifelib-dev/lifelib/blob/f5fe26c2577488683481c376900f18e95b1e2575/lifelib/projects/ifrs17sim/model/IFRS/__init__.py#L308-L309)

Any details on what they relate to? (if public documents)

Thanks
Declan

**Comment by fumitoh** [comment]
These are paragraph numbers in the IFRS17 standard. Note this model is for illustrative purpose, and lots of things are simplifed.
